### DISCUSSION TITLE
Image label incorrect

### DISCUSSION THREAD

**Discussion by human705** [photoprism/photoprism#4429] [Q&A]
Hi All,
I'm using photoprism:latest on ubuntu on docker. I'm adding jpeg pictures from iCloud and i'm trying to tag them so I can search based on keywords. I've imported the attached pictures multiple time and i'm trying to understand why does it get a keyword "basketball". Any help will be greatly appreciated.
TIA
Peter
![IMG_1077](https://github.com/user-attachments/assets/d52c4616-0337-4f05-8d72-32f84598c4af)

**Comment by omerdduran** [comment]
You have the flexibility to remove tags or add new ones that better fit your needs.

https://docs.photoprism.app/user-guide/organize/labels/#add-labels-to-photos
